katas/content/qec_shor/index.md

```diff
@@ -17,48 +17,48 @@ This kata introduces you to the basic concepts of quantum error correction using
 
 **What you should know to start working on this kata:**
 
-- Basic single-qubit and multi-qubit gates
-- Single-qubit and multi-qubit quantum measurements and their effect on quantum systems
+- Basic knowledge of single-qubit and multi-qubit gates
+- Basic knowledge of single-qubit and multi-qubit quantum measurements and their effect on quantum systems
 
 @[section]({
     "id": "qec_shor__noise",
     "title": "Noise in Classical and Quantum Systems"
 })
 
 Any quantum system we can use to carry out quantum computation is inherently noisy. 
-Quantum noise can be caused by different physical processes, depending on the type of a particle or device used as a qubit.
-From the computation point of view, the presence of noise in the quantum system means that its state can suffer from random errors, and thus end up different from the state we're relying on to do our computations. 
+Quantum noise can be caused by different physical processes, depending on the type of particle or device used as a qubit.
+From the computation point of view, the presence of noise in the quantum system means that its state can suffer from random errors, and thus end up in a different state from the state you're relying on to do your computations. 
 This makes computations unreliable very fast, since the effects of noise accumulate quickly to make computation results effectively random.
 
-Before we dive into dealing with noise in quantum systems, let's take a quick look at how we do that for classical systems.
+Before diving into how to deal with noise in quantum systems, let's take a quick look at how to do that for classical systems.
 
-The model used for analyzing classical noise is called *binary symmetric channel*, in which classical bits sent through the channel are transmitted correctly with probability $1-p$ and flipped with probability $p$.
-In this scenario, the information sent through the channel can be protected against the effects of the noise using the *repetition code*:
+The model used for analyzing classical noise is called *binary symmetric channel*. In this model, the classical bits sent through the channel are transmitted correctly with probability $1-p$ and flipped with probability $p$. The error introduced if the bit is flipped is called a *bit flip error*.
+In this scenario, the information sent through the channel can be protected against the effects of the noise, that is, bit flip errors, using the *repetition code*:
 
-- On the sender side, we replace each bit we want to send with three copies of itself:
+- On the sender side, for each bit you create and send three copies of itself:
```

Comment:
Then it has to be rewritten further, since you don't create bits, you have a message in mind that you want to send, and you create copies
"for each bit you want to send, you create and send three copies of it:"